### DISCUSSION TITLE
새 농구장 추가시에 서버에러가 뜹니다😭

### DISCUSSION THREAD

**Discussion by manudeli** [slamapp/server#4] [Q&A]
### 관련 API
courts/new

@sds1vrk 데이터형식에 맞게 POST 요청했는데 에러가 뜨네요 무언가 변경점이 있나요?

관련 이슈 남겨놓아요
https://github.com/slamapp/client/issues/100#event-6060215525

### request POST
```json
{
  "longitude": 127.06859844324877,
  "latitude": 37.744678094850535,
  "image": null,
  "texture": null,
  "basketCount": 1,
  "name": "test 농구장"
}
```

### response
```json
{
  "message": "Server Error",
  "status": 500,
  "errors": []
}
```

**Comment by sds1vrk** [comment]
마누 늦게 답장 드려 죄송해요, 이제야 확인했네요, 로컬 서버로 돌렸을시 에러 인지 아니면 배포된 서버에서의 에러인지 확인 가능할까요?

**Reply by manudeli** [reply]
@sds1vrk 

에러는 로컬서버에서 당시 디벨롭 브랜치에서 난 에러였어요.

이번에 디벨롭 브랜치에 이 API 업데이트가 있나요?

스웨거나온 게 있으면 문서대로 할 수 있으면 좀 더 수월 할 것 같은데 스웨거 API문서가 나왔나요?
